opentok/opentok.py

```diff
@@ -1,17 +1,17 @@
 from datetime import datetime  # generate_token
```

Comment:
There's a ton of changes in here not revolving around issue, can we back these out? Most of them are whitespace/quote changes, but it severely clutters up the PR and should be done as a bespoke cleanup PR.